README.md

```
@@ -1,75 +1,75 @@
+> This starter template has been configured with `v3` version of theme. Due to the limitation of Go module, once the `v4` or up version of theme is released, you need to update the theme manually. (Modifying `config/module.toml` file)
+
+## Deploy to another static page hostings
+
+If you want to build this site using another static page hosting, you need to make sure they have Go installed in the machine. 
+
+<details>
+  <summary>Vercel</summary>
+  
+You need to overwrite build command to install manually Go:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)